bootstrap/eks/main.go

```diff
@@ -51,6 +52,7 @@ func init() {
 	klog.InitFlags(nil)
 
 	_ = clientgoscheme.AddToScheme(scheme)
+	_ = clusterv1.AddToScheme(scheme)
```

Comment:
What's the rationale for adding this? 
I thought it was mentioned previously that this was probably not needed?

Reply:
Definitely needed - the controller looks up Cluster objects